docs/docs/overview/quickstart/get-started-with-python.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 10
+sidebar_label: Get started with Python
+---
+
+# Python Quickstart
+
+Welcome to the Python Quickstart guide for Novu, a powerful notification service that enables you to send multi-channel (SMS, Email, Chat, Push) notifications from your Python applications. In this Quickstart, you'll learn how to seamlessly integrate Novu into your Python project and perform various essential tasks. Let's get started!
+
+## Prerequisites
+
+Before diving into the Quickstart, make sure you have the following:
+
+- Python3 installed on your development machine.
+- A Novu account. If you don't have one, sign up for free at [web.novu.co](https://web.novu.co)
+
+You can also [view the completed code](https://web.novu.co) of this quick start in a GitHub repo.
```

Comment:
Not sure why this link is here. Have you made any demo app for this?

Reply:
@sumitsaurabh927 No, currently I don't made any demo repo. I'll made and update this.